pages/04.Reference/07.dns/docs.en.md

```diff
@@ -27,7 +27,9 @@ The DNS service is shared between all regions. You will need to choose a region
 | Zone transfer to different projects     | yes            | yes
 | Secondary Zones                         | yes            | yes
 | Zone Import / Export                    | yes            | yes
-| PTR records for Floating IPs            | no             | no
+| PTR records for Floating IPs            | yes*           | yes*
+
+*Using the Neutron DNS Integration (more information in the section <a href="#ptr-records-for-floating-ips">PTR records for Floating IPs</a>)
```

Comment:
integration lower-case (to be consistent to the other appearances)

Reply:
```suggestion
*Using the Neutron DNS integration (more information in the section <a href="#ptr-records-for-floating-ips">PTR records for Floating IPs</a>)
```